apps/client/src/intros/shared/DefaultIntro.js

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+import LocationIntro from "./LocationIntro";
```

Comment:
Why does this file exits if it only exports the imported file?

Reply:
There was a reason for it, I think 😊
Will refactor if I can't remember, otherwise I'll add comments.